editor.js

```diff
@@ -24,21 +24,23 @@ XKit.extensions.xkit_editor = new Object({
 var script_editor, icon_editor, css_editor, object_editor;
 
 function extension_editor_run() {
-
+	
+	var mac = navigator.platform.match(/(Mac|iPhone|iPod|iPad)/i) ? "Meta" : "Ctrl";
```

Comment:
Would probably change `mac` to `key`, or `keyText` to be more descriptive.

The `.match()` on platforms other than Mac is probably superfluous, and iOS devices don't have a `meta` key, much less `ctrl` or other modifier keys as far as I know.